alws/crud/distribution.py

```diff
@@ -104,41 +106,66 @@ async def add_distributions_after_rebuild(
             await pulp_client.create_rpm_publication(repo)
 
 
+async def get_existing_packages(pulp_client: PulpClient,
+                                repository: models.Repository):
+    return await pulp_client.get_rpm_repository_packages(
+        repository.pulp_href,
+        include_fields=['pulp_href', 'artifact', 'sha256', 'location_href',
+                        'arch'])
+
+
+async def get_packages_to_add(
+        pulp_client: PulpClient, build_repo: models.Repository,
+        dist_repo: models.Repository):
+
+    def filter_by_arch(pkgs: typing.List[dict], repo_arch: str):
```

Comment:
I saw the same func somewhere in the project. May be better to move it to some util.py module?)